source/Templates/TeamCitySetup/.teamcity/octopus/main/MonoTestingProject.kt

```diff
@@ -0,0 +1,62 @@
+package octopus.main
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.Project
+import jetbrains.buildServer.configs.kotlin.v2019_2.toId
+import octopus.main.buildtypes.MonoTestBuildType
+
+class MonoTestingProject : Project({
+    val projectName = "Mono Testing"
+    name = projectName
+
+    val buildTypesToRegister = sequence {
+        val items = listOf(
+                ("Amazon Linux 2" to "AmazonLinux2"),
+                ("Ubuntu 16.04 LTS" to "Ubuntu16"),
+                ("open SUSE 15.1" to "openSUSE151"),
+                ("open SUSE 12" to "SUSE12"),
+                ("CentOS 7" to "CentOS7"),
+                ("Fedora" to "Fedora32"),
+                ("Debian" to "Debian911"),
```

Comment:
heh. I just found that this should be `Debian913` over [here](https://github.com/OctopusDeploy/BuildAgentAutomation/blob/master/Debian9/scripts/install-teamcity-agent.sh#L57)

Reply:
fixed

Reply:
oh, it's right for now, but i'll be wrong when the upstream gets fixed